pkg/cloud/cloud_interface.go

```diff
@@ -31,9 +31,8 @@ type Cloud interface {
 	WaitForCloneStatus(taskId string) error
 	GetDiskByName(name string) (disk *Disk, err error)
 	GetDiskByNamePrefix(namePrefix string) (disk *Disk, err error)
+	GetAllDisks(instanceID string) (volumes *models.Volumes, err error)
```

Comment:
```suggestion
	GetAllPVMInstanceDisks(instanceID string) (volumes *models.Volumes, err error)
```
More descriptive name.